plugins/inventory/ntnx_prism_vm_inventory.py

```diff
@@ -154,6 +220,36 @@ def _build_host_vars(self, entity):
                 if vm_ip:
                     break
 
+        if getattr(self, "custom_ansible_host", None) and self.custom_ansible_host.get(
+            "expr"
+        ):
+            lookup = {
+                "cluster": cluster,
+                "cluster_uuid": cluster_uuid,
+                "vm_name": vm_name,
+                "vm_description": vm_description,
+                "vm_uuid": vm_uuid,
+            }
+
+            try:
+                vm_ip = re.sub(
+                    r"\{([^}]+)\}",
+                    self.get_replacement_function(lookup),
+                    self.custom_ansible_host.get("expr"),
+                )
+            except Exception as e:
+                if strict:
+                    self.display.error(
```

Comment:
display error should be in else of `if strict` bcz ansible error will already raise it. We need to display if there is strict is false for info